src/Tribe/Cost_Utils.php

```diff
@@ -104,8 +104,14 @@ public function get_separators() {
 	 * return int|float|string
 	 */
 	public function maybe_replace_cost_with_free( $cost ) {
-		if ( '0' === (string) $cost ) {
-			return esc_html__( 'Free', 'the-events-calendar' );
+
+		switch ( (string) $cost ) {
```

Comment:
Maybe we should use `number_format()`  to avoid having to check for a bunch of different values.
```
$dec_point = false !== strpos( '.',  $cost ) ? '.' : ',';
if ( '0.00' === number_format( $cost, 2, $dec_point ) )
```